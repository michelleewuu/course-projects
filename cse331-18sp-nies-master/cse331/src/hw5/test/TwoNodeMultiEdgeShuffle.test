# Same as TwoNodeMultiEdge.test but shuffles the order 
# of adding edges

# Create a graph
CreateGraph two_nodes_multi_edges_shuffle_graph

# Add two nodes
AddNode two_nodes_multi_edges_shuffle_graph a
AddNode two_nodes_multi_edges_shuffle_graph b

# Add five edges
AddEdge two_nodes_multi_edges_shuffle_graph a b AB1
AddEdge two_nodes_multi_edges_shuffle_graph a a AA
AddEdge two_nodes_multi_edges_shuffle_graph a b AB2
AddEdge two_nodes_multi_edges_shuffle_graph b b BB
AddEdge two_nodes_multi_edges_shuffle_graph b a BA

# Print the nodes in the graph, the outgoing edges from a, 
# and outgoing edges from b
ListNodes two_nodes_multi_edges_shuffle_graph
ListChildren two_nodes_multi_edges_shuffle_graph a
ListChildren two_nodes_multi_edges_shuffle_graph b