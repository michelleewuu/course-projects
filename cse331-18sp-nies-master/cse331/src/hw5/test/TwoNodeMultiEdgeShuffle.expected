# Same as TwoNodeMultiEdge.test but shuffles the order 
# of adding edges

# Create a graph
created graph two_nodes_multi_edges_shuffle_graph

# Add two nodes
added node a to two_nodes_multi_edges_shuffle_graph
added node b to two_nodes_multi_edges_shuffle_graph

# Add five edges
added edge AB1 from a to b in two_nodes_multi_edges_shuffle_graph
added edge AA from a to a in two_nodes_multi_edges_shuffle_graph
added edge AB2 from a to b in two_nodes_multi_edges_shuffle_graph
added edge BB from b to b in two_nodes_multi_edges_shuffle_graph
added edge BA from b to a in two_nodes_multi_edges_shuffle_graph

# Print the nodes in the graph, the outgoing edges from a, 
# and outgoing edges from b
two_nodes_multi_edges_shuffle_graph contains: a b
the children of a in two_nodes_multi_edges_shuffle_graph are: a(AA) b(AB1) b(AB2)
the children of b in two_nodes_multi_edges_shuffle_graph are: a(BA) b(BB)